custom/icds_reports/static/js/angular-services/date-helper.service.js

```diff
@@ -22,11 +22,64 @@ window.angular.module('icdsApp').factory('dateHelperService', ['$location', func
         var formattedMonth = moment(getSelectedMonth(), 'MM').format('MMMM');
         return formattedMonth + ' ' + getSelectedYear();
     }
+    function getCustomAvailableMonthsForReports(selectedYear, selectedMonth, monthsCopy, haveAccessToFeatures) {
+
+        var reportStartDates = {
+            'sdd': new Date(2019, 1),
+            'adolescent_girls': new Date(2019, 3),
+        };
```

Comment:
a bit sad to define this twice, but I think it's fine. an alternative would be pulling out to a const.